package.json

```diff
@@ -12,36 +12,35 @@
     "lint": "eslint --ext js,jsx ."
   },
   "dependencies": {
-    "@babel/runtime": "^7.1.2",
-    "bcrypt": "^1.0.1",
+    "@babel/runtime": "^7.1.5",
+    "bcrypt": "^3.0.2",
```

Comment:
Main change in the version bump is switching from `$2a$` to `$2b$` hashes, but the new version can read old hashes.